src/Mono.Android.Export/CallbackCode.cs

```diff
@@ -26,6 +27,7 @@ class DynamicInvokeTypeInfo
 		// NewArray<T>(T[])
 		static readonly MethodInfo jnienv_newarray                        = GetTArrayToIntPtr<int> (JNIEnv.NewArray<int>);
 		static readonly MethodInfo jnienv_getarray                        = GetIntPtrToTArray<int> (JNIEnv.GetArray<int>);
+		[DynamicDependency ("ToLocalJniHandle", "Android.Runtime.CharSequence", "Mono.Android")]
```

Comment:
I don't understand; the following line is a delegate creation for `CharSequence.ToLocalJniHandle()`, so why is this `[DynamicDependency]` needed at all?  Related, if this *is* needed, then why isn't *every other* `static readonly` field here similarly attributed?